Applications/Spire/Source/KeyBindings/OrderTasksTableViewModel.cpp

```diff
@@ -32,9 +32,8 @@ int OrderTasksTableViewModel::get_column_size() const {
 }
 
 AnyRef OrderTasksTableViewModel::at(int row, int column) const {
-  if(row < 0 || row >= get_row_size() || column < 0 ||
-      column >= get_column_size()) {
-    throw std::out_of_range("The row or column is out of range.");
+  if(row < 0 || row >= get_row_size()) {
+    throw std::out_of_range("The row is out of range.");
   }
   static auto empty_value = AnyRef();
```

Comment:
Remove the use of this static variable, it inhibits copy elision:

https://en.cppreference.com/w/cpp/language/copy_elision